tests/cases/helpers/case_helpers_idd_never_alone.yaml

```diff
@@ -141,6 +141,8 @@ IDDCallCarePartnerHelper:
       Strategy: class_chain
       Id: $PAGE.idd_never_alone_app_call.yes_place_call_button$
   # Device screen updates showing you the waiting room.
+    - Type: sleep
```

Comment:
Is there a reason not to use a wait_for  instead of sleep type?

Reply:
It already has a wait_for, with a standard timeout. this is where it was failing the tests, so I added a sleep, to give it a second to connect.

I can't add a wait_for for the connect part, since sometimes is too fast to even register that element. The app is supposed to be fast enough so it doesn't need the sleep, but that's not always the case.